Hangman/Hangman/main.swift

```diff
@@ -7,6 +7,191 @@
+
+var cpuWord = "", hiddenWord = "", updatedWord = "", currentWord = ""
+var bodyPartsLeft = 6, index = 0, updatedBodyParts = 6
+var guess = true, entered = false, guessedRight = true, gameOver = false
+
+let lastIndex = allTheWords.count - 1, firstIndex = 0
+let randomIndex = Int.random(in: firstIndex...lastIndex)
+let randomWord = allTheWords[randomIndex]
+let randomWordArray = randomWord.components(separatedBy: "")
+let blankSpace = "_"
+
+var alphabet = "a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, u, v, w, x, y, z"
+var currentAlphabet = ""
+
+//Intro phrases
+let greeting = "Welcome to Hangman, where if you do not guess the right word in time...YOU HANG!"
+let hangmanPicking = "Wait while the HangMan chooses a word..."
+let explanation = "Guess a letter that's in the word till' the word is complete to win, guess wrong \(bodyPartsLeft) times and you're hanged!"
+var chancesLeftScript = "You have \(bodyPartsLeft) body parts left"
+
+cpuWord = randomWord
+
+//See if can refactor this to use .contains()
+func guessOutput(enteredLetter : String) -> Bool   {
+    if cpuWord.contains(enteredLetter) {
+            guess = true
+    }
+        else {guess = false}
+    
+    return guess
+}
+
+func updateRight(enteredLetter : String) -> String {
+    for char in cpuWord {
+        if char == Character(enteredLetter)    {
+            let startIndex = hiddenWord.startIndex
+            let offset = hiddenWord.index(startIndex, offsetBy: index)
+            hiddenWord.remove(at: offset)
+            hiddenWord.insert(Character(enteredLetter), at: offset)
+            updatedWord = hiddenWord
+            index += 2
+        }
+        else    {index += 2}
+    }
+    index = 0
+    return updatedWord
+}
+
+func updateWrong() -> String {
+    updatedWord = hiddenWord
+    return updatedWord
+}
+
+func enteredLetterBucket(enteredLetter : String) -> [String]   {
+    enteredLetters.append(enteredLetter)
+    let noDuplicates = Set(enteredLetters)
+    enteredLetters = Array(noDuplicates).sorted()
+    return enteredLetters
+}
+
+func checkAlreadyEntered(enteredLettersArray : [String], enteredLetter : String) -> Bool  {
+    if enteredLettersArray.contains(enteredLetter)  {
+        entered = true
+    }
+    else    {entered = false}
+    return entered
+}
+
+func remainingAlphabet(enteredLetter : String, alph : String) -> String    {
+    if alph.contains(enteredLetter) {
+        currentAlphabet = alph.replacingOccurrences(of: enteredLetter, with: " ")
+    }
+    return currentAlphabet //string, contains, removeoccurrences
+}
+
+func checkWordComplete(checkWord : String, soFarWord : String) -> Bool    {
+    var count = 0
+    var complete = false
+    for char in checkWord {
+        if soFarWord.contains(char)   {
+            count += 1
+        }
+    }
+    if count == checkWord.count {
+        complete = true
+    }
+    else {complete = false}
+    return complete
+}
+
+func updateBodyParts(guess : Bool) -> Int  {
```

Comment:
Remember that for full credit, you were to display the picture of the hangman, the updateBodyParts warning was pretty fun though!